src/config.json

```diff
@@ -91,7 +93,7 @@
         "prepush": "npm run eslint",
         "postinstall": "grunt install",
         "test": "grunt test cla-check-pull",
-        "eslint": "grunt eslint"
+        "eslint": "grunt"
```

Comment:
Why remove eslint here?

Reply:
I just fixed this now. I was having some trouble running the script a while ago. But I've since gotten everything working, and forgot to enable it again.